app/views/teams/show.html.erb

```diff
@@ -18,6 +18,33 @@
   <dd><%= @team.description %></dd>
 </dl>
 
+<table class="table table-striped">
+  <thead>
+    <tr>
+    </tr>
+  </thead>
+  <tbody>
+    <% @team.members.each do |member| %>
+      <tr>
+        <td>
+            <%= member.id%> <%= member.first_name%> <%= member.last_name%>
+            
+            <% if @team.owners.include? member %>
```

Comment:
Is it possible to check for the created ability `assign_ownership` in the view? See https://github.com/CanCanCommunity/cancancan/wiki/Checking-Abilities.

Reply:
you could -- but these authorizations logically overlap and are thus are semantically connected to these buttons. you actually _want_ to check, whether a team member is a user nor nawt.